getting_started/access_your_services.adoc

```diff
@@ -180,17 +223,94 @@ Endpoints:                <none>
 Session Affinity:         None
 External Traffic Policy:  Cluster
 ----
+endif::[]
+
+ifdef::openshift-arho[]
+[[expose-tcp-services]]
+== Expose TCP Services
+
+// TODO Need to update this with the Azure-specific load balancer info
+
+{product-title} xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[routes] expose applications by proxying traffic through HTTP/HTTPS(SNI)/TLS(SNI)
+to xref:../architecture/core_concepts/pods_and_services.adoc#architecture-core-concepts-pods-and-services[pods and services]. A link:https://kubernetes.io/docs/concepts/services-networking/#loadbalancer[LoadBalancer]
+service creates an AWS Elastic Load Balancer (ELB) for your {product-title} cluster, enabling direct TCP access to applications exposed by your LoadBalancer service.
+
+[NOTE]
+====
+LoadBalancer services require an additional purchase. Contact your sales team if you are interested in using LoadBalancer services for your {product-title} cluster.
+====
+
+=== Check your LoadBalancer Quota
+
+By purchasing LoadBalancer services, you are provided with a quota of LoadBalancers available for your {product-title} cluster.
+
+----
+$ oc describe clusterresourcequota service-loadbalancers
+Name:       service-loadbalancers
+Labels:     <none>
+Annotations:    <none>
+Resource        Used    Hard
+--------        ----    ----
+services.loadbalancers  0   4
+----
+
+=== Expose TCP Service
+
+You can expose your applications over an external LoadBalancer service, enabling access over the public Internet.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=lb-service
+service/lb-service exposed
+----
+
+=== Create an Internal-Only TCP Service
+
+You can alternatively expose your applications internally only, enabling access only through AWS VPC Peering or a VPN connection.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -
+service/internal-lb exposed
+----
+
+=== Use your TCP Service
+
+Once your LoadBalancer service is created, you can access your service by using the URL provided to you by {product-title}.
+The `LoadBalancer Ingress` value is a URL unique to your service that remains static as long as the service is not deleted.
+If you prefer to use a custom domain, you can create a CNAME DNS record for this URL.
+
+----
+$ oc describe svc lb-service
+Name:                     lb-service
+Namespace:                default
+Labels:                   app=httpd-example
+Annotations:              <none>
+Selector:                 name=httpd-example
+Type:                     LoadBalancer
+IP:                       10.120.182.252
+LoadBalancer Ingress:     a5387ba36201e11e9ba901267fd7abb0-1406434805.us-east-1.elb.amazonaws.com
```

Comment:
azure named loadbalancer
you should probably go through that example and make new output for things like this

Reply:
@grantomation, can you provide the new output for the Azure Load Balancer? Or maybe we can just link to Azure docs?

Reply:
It is difficult to find context for this as the commits are missing. If I go to the renedered version of @dhoyle-rh's branch I can see a few things on this page that require attention. I have put my comments in the google doc that I emailed around yesterday. 

If the loadbalancer service is meant to be created from the command `$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/azure-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -` in the previous section "Create an Internal-Only TCP Service", then the name is wrong and this command should be `oc describe svc internal-lb` instead. If that is the case, then the output will look something like follows;

```
$ oc describe svc internal-lb
Name:                     internal-lb
Namespace:                cake
Labels:                   app=cakephp-mysql-persistent
                          template=cakephp-mysql-persistent
                          template.openshift.io/template-instance-owner=3a53eabd-6b11-11e9-948c-000d3ab9a700
Annotations:              service.beta.kubernetes.io/azure-load-balancer-internal=0.0.0.0/0
Selector:                 name=cakephp-mysql-persistent
Type:                     LoadBalancer
IP:                       172.30.166.170
LoadBalancer Ingress:     52.155.223.79
Port:                     <unset>  8080/TCP
TargetPort:               8080/TCP
NodePort:                 <unset>  30823/TCP
Endpoints:                10.128.2.9:8080
Session Affinity:         None
External Traffic Policy:  Cluster
Events:
  Type    Reason                Age   From                Message
  ----    ------                ----  ----                -------
  Normal  EnsuringLoadBalancer  24m   service-controller  Ensuring load balancer
  Normal  EnsuredLoadBalancer   24m   service-controller  Ensured load balancer
```

However there are a couple of differences. I have used the name "internal-lb" as was created by the previous command and I am using the cake-php example in a project/namespaces that is not "default". We probably don't need to include the events. I hope this helps.